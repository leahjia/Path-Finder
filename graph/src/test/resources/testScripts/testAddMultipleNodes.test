# DesignMap adding nodes testing file.
# Tests that nodes can be added to the map

# Create a graph
CreateGraph map1

# Add a pair of nodes
AddNode map1 n1
AddNode map1 n2
AddNode map1 n3
AddNode map1 n4
