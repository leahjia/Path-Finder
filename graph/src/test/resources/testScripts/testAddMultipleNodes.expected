# DesignMap adding nodes testing file.
# Tests that nodes can be added to the map

# Create a graph
created graph map1

# Add a pair of nodes
added node n1 to map1
added node n2 to map1
added node n3 to map1
added node n4 to map1
